.buildkite/release.steps.yaml

```diff
@@ -88,3 +88,17 @@ steps:
     concurrency: 1
     concurrency_group: "unrealgdk-release"
     <<: *common # This folds the YAML named anchor into this step. Overrides, if any, should follow, not precede.
+
+ # Stage 5: Mirror the release code from Github to Gitee
+  - block: "Unblock mirror code to gitee"
```

Comment:
This block step is not strictly necessary as you explicitly define `depends_on` below. What's the reasoning for its inclusion?

Reply:
I'm not sure whether someone may run this pipeline as test purpose. So I add a block here to avoid the sync operation frequently.
But you are right, it is not strictly necessary that depends_on and block exist as the same time. 
So I choose another option, I have removed the depends_on field from the yaml file, only keep the block step.

Reply:
Ah, Add the depend_on field back because the mirror step must be executed after release step.